src/main/java/Duke.java

```diff
@@ -1,10 +1,218 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+    public static final String MARK_INSTRUCTION = "Please give your command in the following format";
+    public static final String INVALID_INPUT = "This is an invalid input";
+    public static final String HorizontalLine = "____________________________________________________________";
+    public static final String HELLO_IM_DUKE = "Hello! I'm Duke";
+    public static final String KEYWORD_TO_SEE_THE_INSTRUCTIONS = "Please type 'help' if you want to see the instructions";
+    public static final String HELP_USAGE = "help: to view the instructions for all commands\n";
+    public static final String BYE_USAGE = "bye: to end the program\n";
+    public static final String LIST_USAGE = "list: to view the list of tasks\n";
+    public static final String MARK_USAGE = "mark: to mark a task as done";
+    public static final String MARK_FORMAT = "Format: mark {Number}\n";
+    public static final String UNMARK_USAGE = "unmark: to unmark a task as not done yet";
+    public static final String UNMARK_FORMAT = "Format: unmark {Number}\n";
+    public static final String TODO_USAGE = "todo: to add a todo task in your list";
+    public static final String TODO_FORMAT = "Format: todo {your task}\n";
+    public static final String DEADLINE_USAGE = "deadline: to add a deadline task in your list";
+    public static final String DEADLINE_FORMAT = "Format: deadline {your task} /by {deadline date}\n";
+    public static final String EVENT_USAGE = "event: to add an event task in your list";
+    public static final String EVENT_FORMAT = "Format: event {your task} /from {begin date} /to {end date}\n";
+    public static final String ADDING_TASK = "Got it. I've added this task:";
+    public static final String BIG_MARK_NUMBER = "The task number is bigger than the number of tasks";
+    public static final String FINISH_UNMARKING_MESSAGE = "Ok! I've marked this task as not done yet:";
+    public static final String FINISH_MARKING_MESSAGE = "Nice! I've marked this task as done:";
+    public static final String ASKING_MESSAGE = "What can I do for you?";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        showWelcomeMessage();
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        while (true) {
+            userInput = getInput(in);
+            try {
+                String[] seperatedKeywordAndContent = userInput.split(" ", 2);
```

Comment:
Perhaps it would be more readable by adding 2 additional variables to make the keyword and content clearer:

```
String keyword = seperatedKeywordAndContent[0];
String content = seperatedKeywordAndContent[1];
```